.editorconfig

```diff
@@ -135,6 +135,9 @@ csharp_space_between_method_declaration_parameter_list_parentheses = false
 csharp_space_between_parentheses = false
 csharp_space_between_square_brackets = false
 
+# CA2208: Instantiate argument exceptions correctly
+dotnet_diagnostic.CA2208.severity = error
```

Comment:
Sorry for not being precise enough. This only affects VS, but not the build on the command line. Add the warning to the file `src/StyleCop.Analyzers.ruleset` instead. Add a line with `<Rule Id="CA2208" Action="Error" />`.

Reply:
No worries, should be ok now.